### PR TITLE
Use symbols to represent filetypes instead of strings.

### PR DESCRIPTION
**Pull request by woodruffw**
Low priority, just a nice-to-have in a future release.

cc @UniqMartin 
ref https://github.com/Homebrew/brew/pull/378#issuecomment-226939348

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ executables, dynamic libraries, and so forth.
 
 ### Documentation
 
+**Important**: ruby-macho does not have a stable API (yet). Use it with this
+in mind.
```
Comment (UniqMartin) [inline README.md]: Good call!

**Comment by UniqMartin** [comment]
:+1:

**Comment by UniqMartin** [comment]
Not sure if you still want to look into something as this is marked as `in progress`, but feels ready to me. A lone :+1: or one followed by some affirmative comment is meant as “merge at your convenience”.

**Comment by woodruffw** [comment]
Oh, no, I just hadn't gotten around to actually hitting the merge button yet :sweat_smile:.

**Comment by UniqMartin** [comment]
Aye, no pressure! Just wanted to make sure there's no accidental misunderstanding. 😀

**Comment by woodruffw** [comment]
No worries!

**Pull request merged**